### DISCUSSION TITLE
Request for the Addition of a Old/New Feature - fn Key to Turn On/Off Screen

### DISCUSSION THREAD

**Discussion by lspss60309** [seerge/g-helper#1075] [General]
Dear Serge,

First and foremost, I would like to extend my gratitude for the exceptional quality and innovation that your products consistently embody. It is evident that your team is dedicated to providing solutions that cater to the needs of your users.

The feature I am referring to is the ability to use the fn key to turn off/on the screen. Like Old ASUS laptop fn+f7 feature.

I understand that each feature addition requires careful consideration and evaluation, so I'm just presenting an idea that I hope could be discussed for potential incorporation in your future product development. Thank you very much for your time, and I look forward to hearing from you.

Wishing your team continued success,

Andy
lspss60309@gmail.com

**Comment by seerge** [comment]
@lspss60309 what kind of laptop do you have ? on modern devices FN+F7/F8 control brightness

**Reply by lspss60309** [reply]
@seerge ASUS K43SD.  it can press fn+f7 to turn on/off the laptop monitor.
If it's not feasible, I would require a similar function as depicted below:
fn+xxxKey to turn off monitor temporary, lights up when you press key or move mouse. (←It is like the function in a windows app called  Twinkle Tray.  Here is an accompanying image for reference)
![twinkle-tray function](https://github.com/seerge/g-helper/assets/36922194/0689f4a2-7f19-4de2-9d64-a122e2b163e0)
And Here is an official link of ASUS K43SD support:
https://www.asus.com/tw/supportonly/k43sd/helpdesk_download/
Somehow, when I switch the URL to English, I can't find the product.
Please translate the webpage into English.

**Reply by seerge** [reply]
If F7 turns of the screen for you, then i assume rest of the buttons are also don't match? 

Just in case you van already bind Toggle screen to a M1-M4 via Extra

**Reply by lspss60309** [reply]
I apologize for not making it clear earlier.
I have two laptops, an old ASUS K43SD and a new GA401QM (G14). [G14 can not use fn+anykey to turn off the screen]
I want to enable the turn off screen function on my G14, similar to the K43SD (fn+f7).

**Reply by seerge** [reply]
You already can do that via Extra menu as i have mentioned:)

**Reply by lspss60309** [reply]
I'll try it later. Thank you very much!

**Reply by lspss60309** [reply]
I've found this feature!
I really appreciate your help. Thank you.
Sorry for taking up your time.
I initially thought the function was like Win+P, this was a misunderstanding due to the translation.
